### DISCUSSION TITLE
Defaulting to Dark Mode on Initial Load in VitePress

### DISCUSSION THREAD

**Discussion by eyallampel1** [vuejs/vitepress#2895] [Q&A] [answered]
Hello everyone,

I've built a VitePress site it has a toggle for dark/light mode. Currently, the site loads in light mode by default. I'd like to set dark mode as the initial theme when users visit the site. They should then be able to switch to light mode manually if they choose to.

Could anyone guide me on how to set dark mode as the default theme on initial load? Any pointers would be greatly appreciated!

Thank you in advance!

**Comment by brc-dd** [comment, marked as answer]
Set `appearance: 'dark'` in your config - https://vitepress.dev/reference/site-config#appearance

**Reply by eyallampel1** [reply]
Thanks

**Reply by MikeCZ23** [reply]
how? where?
